doc/stdenv/stdenv.chapter.md

```diff
@@ -1109,10 +1109,17 @@ This setup hook moves any libraries installed in the `lib64/` subdirectory into
 
 This setup hook moves any systemd user units installed in the `lib/` subdirectory into `share/`. In addition, a link is provided from `share/` to `lib/` for compatibility. This is needed for systemd to find user services when installed into the user profile.
 
+This hook only runs when compiling for Linux.
+
 ### `set-source-date-epoch-to-latest.sh` {#set-source-date-epoch-to-latest.sh}
 
 This sets `SOURCE_DATE_EPOCH` to the modification time of the most recent file.
 
+### Other hooks
+
+Many other packages provide hooks, that are not part of `stdenv`. You can find
+these in the [Hooks Reference](#chap-hooks).
+
```

Comment:
This should be at the bottom IMO.

Reply:
Now it comes right after the last hook. I don't think moving it to the bottom is an improvement, because someone who is looking for hooks is unlikely to read past the section below, as they will assume that all hooks have been discussed at that point.

Reply:
Oh, they are hooks. I'll fix it.